# HG changeset patch
# User Max Brister <max@2bass.com>
# Date 1340118887 18000
#      Tue Jun 19 10:14:47 2012 -0500
# Node ID 9d60ee38c6e28a4ced90d9174cd988dd3ba38668
# Parent  466cb8673653902599c8552b3ad2c5c9ae54e7ee
pt-jit.h: Remove unused variables from jit_block

diff --git a/src/pt-jit.h b/src/pt-jit.h
--- a/src/pt-jit.h
+++ b/src/pt-jit.h
@@ -1312,20 +1312,16 @@ private:
   size_t mid;
   jit_block *idom;
   df_set mdf;
   std::vector<jit_block *> dom_succ;
   std::string mname;
   instruction_list instructions;
   bool malive;
   std::list<jit_block *>::iterator mlocation;
-
-  jit_phi_incomming *use_head;
-  jit_phi_incomming *use_tail;
-  size_t myuse_count;
 };
 
 // keeps track of phi functions that use a block on incomming edges
 class
 jit_phi_incomming : public jit_internal_node<jit_block, jit_phi_incomming>
 {
 public:
   jit_phi_incomming (void) {}
